OsmAnd/src/net/osmand/plus/myplaces/MyPlacesActivity.java

```diff
@@ -48,9 +55,11 @@ public class MyPlacesActivity extends TabActivity {
 	private OsmandSettings settings;
 
 	private ViewPager viewPager;
-	private List<WeakReference<FragmentStateHolder>> fragmentsStateList = new ArrayList<>();
+	private final List<WeakReference<FragmentStateHolder>> fragmentsStateList = new ArrayList<>();
 	private int tabSize;
 
+	private final List<ActivityResultListener> activityResultListeners = new ArrayList<>();
```

Comment:
Probably we could try to move it to upper level & remove copy paste with mapActivity